common/log/tag/values.go

```diff
@@ -144,6 +144,9 @@ var (
 	ComponentActiveClusterManager             = component("active-cluster-manager")
 	ComponentMembershipResolver               = component("membership-resolver")
 	ComponentHashring                         = component("hashring")
+	ComponentNamespaceManager                 = component("namespace-manager")
```

Comment:
let's call this shard-namespace-manager. namespace is too generic and can be confused with domains for folks coming from other similar technologies

Reply:
Good point. Added a shard- prefix and also renamed Namespace to ShardNamespace.

Sidenot: I'm still unsure how these parts will be moved outside to Cadence, since this is super Cadence specific.